common/gnome-shell/3.18/sass/_common.scss

```diff
@@ -2308,15 +2324,13 @@ $legacy_icon_size: 24px;
 
 .screen-shield-notification-count-text { padding: 0px 0px 0px 12px; }
 
-#panel.lock-screen { background-color: transparentize($_bubble_bg_color, 0.5); }
-
```

Comment:
While I understand that it makes sense to move this with the rest of the `.lock-screen` styling. It's best to keep the stylesheet structure identical to the upstream code. This is for keeping the theme maintenance simple.

For that reason I'd prefer to keep this rule at its current locations.